src/Uno.UI/UI/Xaml/UIElement.Skia.cs

```diff
@@ -29,10 +26,10 @@ public partial class UIElement : DependencyObject
 		internal double _canvasLeft;
 		private Rect _currentFinalRect;
 
-		private protected int? Depth { get; private set; }
-
 		public UIElement()
 		{
+			_isFrameworkElement = this is FrameworkElement;
```

Comment:
Codacy found an issue: [Offload the code that's conditional on this 'is' test to the appropriate subclass and remove the test.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5943777)